calicoctl.py

```diff
@@ -595,7 +595,7 @@ def node_show(detailed):
                     for ep_id, endpoint in endpoints.iteritems():
                         x.add_row([host,
                                    container_type,
-                                   workload,
+                                   workload[:12],
```

Comment:
I'd prefer a static var here, or even just a comment explaining where 12 comes from. Not immediately obvious from looking at the code (though I believe it's what docker does?)